tests/recipient_validation/test_phone_number.py

```diff
@@ -252,52 +243,59 @@ def test_get_international_info(phone_number, expected_info):
         pytest.param("(1)2345", marks=pytest.mark.xfail),
     ],
 )
-def test_normalise_phone_number_raises_if_unparseable_characters(phone_number):
-    with pytest.raises(InvalidPhoneError):
-        normalise_phone_number(phone_number)
+def test_instantiating_phonenumber_raises_if_unparseable_characters(phone_number):
+    with pytest.raises(InvalidPhoneError) as error:
+        PhoneNumber(phone_number)
+    assert str(error.value) == InvalidPhoneError.ERROR_MESSAGES[InvalidPhoneError.Codes.UNKNOWN_CHARACTER]
 
 
 @pytest.mark.parametrize(
     "phone_number",
     [
-        "+21 4321 0987",
+        "+21 4321 0987",  # region north africa +21 alone isn't a valid country code
         "00997 1234 7890",
         "801234-7890",
         "(8-0)-1234-7890",
     ],
 )
 def test_get_international_info_raises(phone_number):
     with pytest.raises(InvalidPhoneError) as error:
-        get_international_phone_info(phone_number)
-    assert str(error.value) == InvalidPhoneError.ERROR_MESSAGES[InvalidPhoneError.Codes.UNSUPPORTED_COUNTRY_CODE]
+        number = PhoneNumber(phone_number)
+        number.get_international_phone_info()
```

Comment:
I wonder, does this test raise on calling this line, or on instantiation? My hypothesis is the test might still work without line 264? But then of course we'd need to rename it - maybe to something like `test_PhoneNumber_raises_for_invalid_international_numbers()`?